cppapi/client/event.cpp

```diff
@@ -1837,7 +1837,14 @@ void Tango::EventConsumer::initialize_received_from_admin(const Tango::DevVarLon
                                                           const string &adm_name,
                                                           bool device_from_env_var)
 {
- 	long server_tango_lib_ver = (dvlsa->lvalue.length() > 0) ? dvlsa->lvalue[0] : 0;
+ 	if(dvlsa->lvalue.length() == 0)
```

Comment:
After deeper investigations and tests, it appears that this modification makes it impossible for Tango 9.3.1 device servers to subscribe to Tango <= 8 device servers. This is definitely a too strong limitation.
We need to change that.

Reply:
@bourtemb , we will need a docker with Tango 7 server for that - it is very hard to fix this blindly, I will be back in office next week and can start to prepare one... Hope this does not delay you too much!